bindings/wasm/src/did/wasm_document.rs

```diff
@@ -153,6 +313,39 @@ impl WasmDocument {
     method.revoke_merkle_key(index).wasm_result()
   }
 
+  /// Attaches the relationship to the given method, if the method exists.
+  ///
+  /// Note: The method needs to be in the set of verification methods,
+  /// so it cannot be an embedded one.
+  #[wasm_bindgen(js_name = attachMethodRelationships)]
+  pub fn attach_method_relationships(
```

Comment:
This should be singular after the latest changes.

```suggestion
  #[wasm_bindgen(js_name = attachMethodRelationship)]
  pub fn attach_method_relationship(
```